.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,50 @@
+name: 🐛 Bug
+description: Report that a package is broken or something is not working as expected
+labels: [":bug: bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting that something is broken in [VM-Packages](https://github.com/mandiant/VM-Packages). Please ensure that there is no [issue](https://github.com/mandiant/VM-Packages/issues?q=is%3Aopen+is%3Aissue+label%3A%22%3Anew%3A+package%22) already for this bug. If you know how to solve this problem, please send a pull request! :pray:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What's the problem?
+      description: Include the actual and expected behavior. The more details, the better!
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce
+      placeholder: |
+        1. First Step
+        2. Second Step
+        3. and so on…
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Include details about your environment such as your host OS, VM OS, virtualization software, VM hostname and Powershell version in the VM.
```

Comment:
I am not sure what information we need. @mandiant/flare-vm suggestions/feedback on current proposal?

Reply:
this looks like a good start

Reply:
maybe we can create a `Get-VM-Info` command that users can run and copy-paste?

Reply:
> I would also avoid the host OS and virtualization software since we're only familiar with VMWare and probably don't want to try to triage non-VMWare issues.

I use FLARE-VM using virtualbox as well 😶  Even if we decide not to triage some issues, it may still be useful to know the host OS and virtualization software, as we may not be able to reproduce these bugs.

Reply:
I have modified this description a bit to include some of the fields from Blaine's list. I think if we want to ask for everything Blaine's suggested, we should create a command as @mr-tz suggested. Otherwise this is too time consuming and will discourage users from reporting bugs. I have created a issue: https://github.com/mandiant/VM-Packages/issues/49